app/Main.hs

```diff
@@ -85,27 +85,27 @@ main =
                 (viewRef, refreshAction) <- runLoggingT
                   (runReaderT refresh (ClientContext config mgr))
                   print
-                -- updater thread
-                forkIO
-                  . forever
-                  $ withFDHandler defaultBatchingOptions logHandle 1.0 80
-                  $ \logToHandle ->
-                      runReaderT (refreshAction upd) (ClientContext config mgr)
-                        `runLoggingT` ( logToHandle
-                                      . renderWithTimestamp
-                                          (formatTime
-                                            defaultTimeLocale
-                                            (iso8601DateFormat
-                                              (Just "%H:%M:%S")
+                Async.concurrently_ 
```

Comment:
Could we add something like `finally` or `bracket` to log when the server is shutdown?